### ISSUE TITLE
high poly object textures and materials

### ISSUE THREAD

**Issue by ilievmark** [xamarin/urho#373]
I have specific task, I hope you can help me)
I have plane with 22000 virtices and i have to set image texture on this one. When i try method FromImage to recieve material that i need, plane seems like color filled.
What can I do with this?

**Comment by EgorBo** [comment]
@DikiyFoxy your plane should have UV coordinates

**Comment by ilievmark** [comment]
@EgorBo Im not good familiar with game engines and 3d modeling
Maybe you have samples or references of similar solution?
I prepare my model in Blender
![screen shot 2019-02-14 at 15 00 37](https://user-images.githubusercontent.com/32593471/52788358-58248b00-3069-11e9-8c13-2803897e29b3.png)
Ive tried doing Smart UV project mapping, but it didnt give necessary effect

**Comment by ilievmark** [comment]
@EgorBo After baking texture with "Normals Bake mode" i receive the same effect  as in the game
![screen shot 2019-02-14 at 15 04 39](https://user-images.githubusercontent.com/32593471/52788564-0a5c5280-306a-11e9-96c9-d02e433bb586.png)

**Comment by ilievmark** [comment]
@EgorBo Thanks for asking. The problem was with one check box in blender settings for object:
![screen shot 2019-02-14 at 21 36 10](https://user-images.githubusercontent.com/32593471/52812816-07c91f80-30a1-11e9-8f51-6213d8e20fb1.png)

**Comment by EgorBo** [comment]
@DikiyFoxy yeah totally forgot about this one, however the CLI tools to convert formats (AssetImporter) by default export UVs afair
